src/Definition/Reflection/Parameter.php

```diff
@@ -63,11 +63,14 @@ public function getPosition() : int
      */
     public function getType() : ?string
     {
-        if ($this->reflection->hasType()) {
-            return $this->reflection->getType()->getName();
+        if (! $this->reflection->hasType()) {
+            return null;
         }
 
-        return null;
+        $type = $this->reflection->getType();
+        assert($type !== null);
```

Comment:
it has no sense

if we are checking first `hasType()` then `getType()` can't be null

Reply:
Guess i was too slow. PHPstan does not infer it correctly. Not sure if an update will change this. Might try this as well-